content/zh/docs/chart_template_guide/wrapping_up.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "下一步"
+description: "收尾 - 一些能帮到你的指向其他的文档的链接。"
+weight: 14
+---
+
+本指南旨在为chart开发者提供对如何使用Helm模板语言的强大理解能力。该模板聚焦于模板开发的技术层面。
+
+但涉及到chart的实际日常开发时，很多内容本指南并没有覆盖到。这里有一些有用的文档链接帮助你创建新的chart:
+
+- [Helm的Chart项目](https://github.com/helm/charts) 是不可或缺的chart资源。这个项目也定义了
```

Comment:
We recently updated this to point to Artifact Hub: https://github.com/helm/helm-www/pull/1016

Reply:
Got it, I'll update this point.